spec/models/event_spec.rb

```diff
@@ -121,4 +121,15 @@
       end
     end
   end
+
+  describe '.name_and_date' do
```

Comment:
This should be `#name_and_date` to signify that it's an instance method.

http://betterspecs.org/#describe